src/fu-util-common.c

```diff
@@ -915,7 +911,7 @@ fu_util_device_to_string (FwupdDevice *dev, guint idt)
 		g_autofree gchar *desc = NULL;
 		desc = fu_util_convert_description (tmp, NULL);
```

Comment:
Perhaps `fu_util_convert_description()` should have done this? I'm not sure why we'd want leading or trailing spaces on the CLI.

Reply:
Yeah I'll move it in function.

Reply:
I guess we should also be using `xb_node_transmogrify()` at some point too, but no panic as it's not yet in any release -- and I think the LVFS can't emit description with inline markup until everything consuming it has support, which probably makes it impossible to actually do.

Reply:
Moved it in function.